Add drake narrative

wlandau opened the pull request:
For ropensci/onboarding#156. The 2018-02-10 date and `topicid` are only placeholders.

stefaniebutland commented on content/blog/2018-02-10-drake.md:
```diff
@@ -0,0 +1,117 @@
+authors:
+  - name: Will Landau
+    url: https://github.com/wlandau
+date: 2018-02-10
```
Leave date as is for now. Tentative publication date is 2018-02-13, but this may be postponed by ~ a week if a priority rOpenSci post must be published - e.g. announcing unconference nominations open

stefaniebutland commented on content/blog/2018-02-10-drake.md:
```diff
@@ -0,0 +1,117 @@
+
+The [drake](https://github.com/ropensci/drake) package is a [pipeline](https://github.com/pditommaso/awesome-pipeline) toolkit. It manages data science workflows, [saves time](https://ropensci.github.io/drake/#what-gets-done-stays-done-), and [ensures a kind of reproducibility](https://ropensci.github.io/drake/#reproducibility-with-confidence) that the R community seldom thinks about. I am trying ...
+
+# Dissertation frustration
```
These headings look big when rendered. Consider ### See example https://ropensci.org/blog/2017/12/05/rperseus/

stefaniebutland commented on content/blog/2018-02-10-drake.md:
```diff
@@ -0,0 +1,117 @@
+    keep_md: yes
+---
+
+The [drake](https://github.com/ropensci/drake) package is a [pipeline](https://github.com/pditommaso/awesome-pipeline) toolkit. It manages data science workflows, [saves time](https://ropensci.github.io/drake/#what-gets-done-stays-done-), and [ensures a kind of reproducibility](https://ropensci.github.io/drake/#reproducibility-with-confidence) that the R community seldom thinks about. I am trying ...
```
name package in backticks like [`drake`](https://github.com/ropensci/drake)

stefaniebutland replied:
I'm concerned about how this part sounds: "...ensures a kind of reproducibility that the R community seldom thinks about. I am trying to change the conversation around reproducible research ..."

The first part isn't necessarily an accurate reflection on the R community as it is today.  This may be your personal experience and that's always a useful viewpoint, however, many people are actively doing and thinking about that and many have been trying to change the conversation for a while. I would consider how to rephrase to make that more clear, taking care not to offend potential users or fans of `drake` before they try it out. Consider it a way to engage with precisely the kinds of people you want to work with and among in order to make the change.

stefaniebutland commented on content/blog/2018-02-10-drake.md:
```diff
@@ -0,0 +1,117 @@
+
+# Dissertation frustration
+
+<center><img src="/img/blog-images/2018-02-10-drake/uphill.png" style = "width: 300px" alt="Sisyphus"></center>
```
Please credit images in text below image. They seem to come from here: https://sites.google.com/site/sisyphusa/

See example img credit https://ropensci.org/blog/2017/11/17/unconf-sixtips/

`<img src="/img/blog-images/2017-11-17-unconf-sixtips/ropensci-unconf17-community-nistara-randawa.jpg" alt="Attendees at the May 2017 rOpenSci unconference. Photo credit: Nistara Randhawa">
_Attendees at the May 2017 rOpenSci unconference. Photo credit: Nistara Randhawa_`

wlandau commented:
@stefaniebutland I am glad you made those comments, and I think I have addressed all your feedback so far. Please let me know if there is anything else I can do.

stefaniebutland commented:
Nicely done @wlandau!
I will be in touch when I can confirm the post date. Thank you again

stefaniebutland commented:
@wlandau I'd like to publish your post Feb 6. Please update post date to 2018-02-06 and other date-dependent things like folder name for images.

wlandau commented:
Fantastic, @stefaniebutland! I just switched all the dates.

wlandau commented:
So amazing to see this post deployed and live! Thank you again for your help.

stefaniebutland commented:
You're very welcome Will. The package really seems to have generated interest and your narrative in the post is fantastic! Nice work, and good luck with its continued development
